guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -41,26 +41,16 @@ endif::[]
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *Register Host*.
-. Optional: Select a different *Organization*.
-. Optional: Select a different *Location*.
-. Optional: From the *Host Group* list, select the host group to associate the hosts with.
-Fields that inherit value from *Host group*: *Operating system*, *Activation Keys* and *Lifecycle environment*.
-+
-If your host group has any Ansible roles assigned, the Ansible roles will run on your host upon the registration.
-. Optional: From the *Operating system* list, select the operating system of hosts that you want to register.
-ifndef::satellite,orcharhino[]
-Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
-endif::[]
-. Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
-+
-[NOTE]
-====
-A {SmartProxy} behind a load balancer takes precedence over a {SmartProxy} selected in the {ProjectWebUI} as the host's content source.
-====
-ifdef::katello,satellite,orcharhino[]
-. In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
-endif::[]
-. Optional: Select the *Insecure* option, if you want to make the first call insecure.
+. Enter the details for how you want the registered hosts to be configured.
+. Click *Generate*.
+. Run the `curl` command as `root` on the host that you want to register.
+After registration completes, any Ansible roles assigned to a host group you specified when configuring the registration template will run on the host.
+
+The registration details that you can specify include the following:
```

Comment:
```suggestion
You can configure the following registration options:
```
If they are all optional, this lead-in sentence means that you can remove "Optional:" from the steps. 

I think steps are easier to ready if they do not begin with "Optional:", so I try to keep their use to a minimum.

Reply:
> If they are all optional, this lead-in sentence means that you can remove "Optional:" from the steps.

However, what if we need to add a required step some time in the future and forget to update the lead-in sentence? In reality (and under pressure) writers sometimes don't read all the surrounding text to make sure it's still valid. So I'd prefer to use a sentence that will work under all circumstances, no matter what kind of option (optional/required) gets added to the list of options. WDYT?

> I think steps are easier to ready if they do not begin with "Optional:", so I try to keep their use to a minimum.

Agreed. I could also just drop all the "Optional:" bits..? But then again, the rest of each sentence would make it sound as if the option is required:

> Optional: On the General tab, in the Capsule field, select the Capsule to register hosts through.

vs

> On the General tab, in the Capsule field, select the Capsule to register hosts through.

Reply:
I kept the original lead-in sentence, but dropped "Optional:" and rephrased each option description to use "you can" instead of a plain imperative. I hope that should address the concern while still keeping the wording precise.